### PR TITLE
these files have not changed in mediawiki

### PR DESCRIPTION
**Pull request by Metallicow**
May 13th 2018. These are the files I pulled out of my wipz project that didnt change on mediawiki since then. comparison with a drag and drop in github 4 windows from my download script. They will help speed up the process of doing missing pages to plug into your structure that may not be done yet. These will need checked as some things probably differ in the way I was doing things. Initial commit of my work so far that doesnt have to be redone.

### CODE REVIEW

File: these_files_didnt_change_5_13_2018/.csv.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+The format consists of a text file with .csv extension, together with a
+folder under the same name and a .frames extension. The .csv file and
+the folder must be on the same path location. The text file contains the
+parameters for the scene, like the field resolution and frame rate, and
+also contains the exposure sheet for the layers. The folder contains
+:doc:`.png` picture files. Unlike image sequences, a key frame
+instance is only a single file and the exposure sheet links it to one or
+more frames on the timeline.
+
+.. figure:: images/csv/Csv_spreadsheet.png
```
Comment (ghost) [inline these_files_didnt_change_5_13_2018/.csv.rst:24]: I may be wrong but is the path for the images correct. The images are in the /images/en/ folder I think

**Comment by therahedwig** [comment]
Of these, the only ones that hadn't been ported yet were the 'Brush-tips_' ones, and those indeed need the images to be fixed and stored in the /images/en folder.

Still, that's at the least a third of the tutorials.

File: these_files_didnt_change_5_13_2018/.csv.rst (hunk 1 above)
Comment (Metallicow) [reply these_files_didnt_change_5_13_2018/.csv.rst:24]: no they will need changed and checked. I didnt add image paths just basic rst

**Comment by scottpetrovic** [comment]
Does this still need a bit of work? There is a folder called "these_files_didnt_change_5_13_2018". It looks like these files should be moved to the correct sections before it would be ready to be merged.

**Comment by Metallicow** [comment]
um yea its just a reference commit. Files can be copied and reviewed and sent as another PR. Then this one can just be closed when all is done. No need to merge this one.  Image paths, File location, etc need changed then reviewed for accuracy. standard fare. Just pick out the ones that still need plugged in and go from there. I'm a bit busy in real life atm with some other projects and wxPy so I just uploaded what you could use to speed up the rest of the porting. Not sure how much time i'll have. We got a whole building to remodel and setup for the new computer refurb shop move. Best if someone else familiar with your style picks them out and finishes them properly.

**Comment by Metallicow** [comment]
It appears that the docs have been fully ported now, so this isnt needed anymore on my end.
Thanks for the port work. Closing this.

**Pull request closed without merging**